From 60fd6c348917c108fccc50bb5a3e8805275eed64 Mon Sep 17 00:00:00 2001
From: Cole Robinson <crobinso@redhat.com>
Date: Fri, 21 Mar 2014 14:27:05 -0400
Subject: [PATCH] spice: input: Fix absolute mouse y coordinates

Current tablet + spice is unusable. Regressed with the UI input rework.

(cherry picked from commit 2445826353f0c65fd9920260f682fb725a36b399)
---
 ui/spice-input.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/spice-input.c b/ui/spice-input.c
index 6dab23b..c342e0d 100644
--- a/ui/spice-input.c
+++ b/ui/spice-input.c
@@ -176,7 +176,7 @@ static void tablet_position(SpiceTabletInstance* sin, int x, int y,
 
     spice_update_buttons(pointer, 0, buttons_state);
     qemu_input_queue_abs(NULL, INPUT_AXIS_X, x, pointer->width);
-    qemu_input_queue_abs(NULL, INPUT_AXIS_Y, y, pointer->width);
+    qemu_input_queue_abs(NULL, INPUT_AXIS_Y, y, pointer->height);
     qemu_input_event_sync();
 }
 
